tests/infrastructure/instance_types/test_cpu_memory_hotplug_instancetype.py

```diff
@@ -86,6 +90,18 @@ def hotplug_vm_snapshot_instance_type(instance_type_hotplug_vm):
         yield snapshot
 
 
+def hot_plug_cluster_instance_type(guest_sockets, cpu_model):
```

Comment:
name is a bit confusing, it sounds like you do the hotplug, but from my understanding here you just prepared new instance type that will be used later for the hotplug process